requirements.txt

```diff
@@ -1,2 +1,3 @@
-PyYAML
-jinja2
+PyYAML==3.12 ; python_version < '3.7'
```

Comment:
3.7 -> 3?

Reply:
Really
It was extracted from odoo/odoo
This version has issues only using py3.7

Why do you think that should be used for all py3 instead?

Reply:
Ok, I though it was a typo.